p/sbb/readme.md

```diff
@@ -0,0 +1,16 @@
+# SBB profile for `hafas-client`
+
+[*Schweizerische Bundesbahnen (SBB)*](https://en.wikipedia.org/wiki/Swiss_Federal_Railways) is the largest Swiss long-distance public transport company. This profile adds *SBB*-specific customizations to `hafas-client`.
+
+## Usage
+
+```js
+const createClient = require('hafas-client')
+const sbbProfile = require('hafas-client/p/sbb')
+
+// create a client with ÖBB profile
+const client = createClient(sbbProfile, 'my-awesome-program')
+```
+
```

Comment:
Let's add a note about how to customize `profile.locale` & `lang` here.

Reply:
what do you think of implementing something like
```
const profile = require('hafas-client/p/sbb').forLocale(locale: 'fr_CH' | 'it_CH' | 'de_CH')
```

Reply:
I like that, from a user perspective, it's a reasonably ergonomic API, and it doesn't interfere with the `createClient` logic in `hafas-client/index.js`. I don't like though that `forLocale` effectively becomes part of the profile passed into `createClient`.

What about having two (three if you include Italian) entrypoints `hafas-client/p/sbb/de-ch` & `hafas-client/p/sbb/fr-ch`? There would be a additional file, but it would just extend `require('./de-ch')`.

Reply:
nice and simple, i like it